pkg/license/identifier.go

```diff
@@ -102,3 +104,23 @@ func Identify(pkgPath, content string) (string, error) {
 		return "", fmt.Errorf("cannot identify license content")
 	}
 }
+
+// GetLicenseContent from license id
+func GetLicenseContent(spdxID string) (string, error) {
+	files, err := assets.AssetDir(licenseTemplatesDir)
+	if err != nil {
+		return "", err
+	}
+	for _, f := range files {
+		if spdxID != strings.TrimSuffix(f.Name(), filepath.Ext(f.Name())) {
+			continue
+		}
+
+		res, err := assets.Asset(filepath.Join(licenseTemplatesDir, f.Name()))
+		if err != nil {
+			return "", err
+		}
+		return string(res), nil
+	}
+	return "", fmt.Errorf("could not found license: %s", spdxID)
```

Comment:
Can we just join the `licenseTemplateDir`, `spdxID` and `.txt` to avoid iterating all the files over and over again?

```suggestion
	res, err := assets.Asset(filepath.Join(licenseTemplatesDir, spdxID, ".txt"))
	if err != nil {
		return "", err
	}
	return string(res), nil
```